nvflare/fuel/utils/config.py

```diff
@@ -59,43 +60,44 @@ def get_native_conf(self):
             return native config objects
         """
 
-        pass
+        return self.conf
 
-    @abstractmethod
     def get_location(self) -> Optional[str]:
         """return the file path where this configuration is loaded from
 
         Returns:
             return None if the config is not from file else return file path
 
         """
-        pass
+        return self.file_path
 
     @abstractmethod
-    def to_dict(self) -> Dict:
+    def to_dict(self, resolve: Optional[bool] = True) -> Dict:
```

Comment:
if only OmegaConf need this "resolve" should we add it into the base class definition?